### ISSUE TITLE
HT.2 (XBB.1.5.49.2) with ORF1a:M3752I (nsp6_M183I) and S:T478R (17 seqs, 4 countries)

### ISSUE THREAD

**Issue by NkRMnZr** [cov-lineages/pango-designation#2275]
Taken from **Branch 7** of multitask sars-cov-2-variants/lineage-proposals#451

**Defining Mutations:** `HT.2` > `G11521T(ORF1a:M3752I/nsp6_M183I), C15960T` > `C22995G(S:T478R)`
**Query:** `G11521T, C21714T, C22995G`
**Earliest seq:** 2023-07-05 (`EPI_ISL_17985212`, Basque, Spain)
**Latest seq:** 2023-08-13 (`EPI_ISL_18222970`, California, USA)
**Sampled Countries:** USA (10: CA/7; HI; NY; OR), Spain (5), France (1), Mexico (1) 

<details><summary>Genomes: </summary>

`EPI_ISL_17985212, EPI_ISL_18044159, EPI_ISL_18049081, EPI_ISL_18060664, EPI_ISL_18072611, EPI_ISL_18073902, EPI_ISL_18094328, EPI_ISL_18095035, EPI_ISL_18108081, EPI_ISL_18121993, EPI_ISL_18137298, EPI_ISL_18222421, EPI_ISL_18222538, EPI_ISL_18222711, EPI_ISL_18222970, EPI_ISL_18233893, EPI_ISL_18238034`</details>

**UShER:**

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_91dd_c5ddf0.json?f_userOrOld=uploaded%20sample&label=id:node_7171398
![image](https://github.com/cov-lineages/pango-designation/assets/125747944/1d34ed77-290d-41c3-9d7b-62b1c32b2187)

**Comment by corneliusroemer** [comment]
Too small and doesn't have novel RBD haplotype either

**Comment by FedeGueli** [comment]
@corneliusroemer i dont think closing things in two days helps a lot. It was choosen cause we saw a high homoplasy level on XBB.1.5.49 especially in Mexico we tracked them all for a while. Backbone is not that fast but this one likely for its geographical spread wasnt that [slow either](https://cov-spectrum.org/explore/World/AllSamples/Past3M/variants?nextcladePangoLineage=EG.5.1.1*&nucMutations1=G11521T%2CC21714T%2CC22995G&analysisMode=CompareToBaseline&). I am not saying we should designate everything but tracking for a bit more what has been selected after a good work of tracking could be more sensitive and maybe also give more fruit.
Also likely you dint notice that this got mutations at 261 and 583 in the follow up branches. but ok.
